Presentation.Web/app/components/org/structure/org-unit-migration-table.component.ts

```diff
@@ -108,6 +115,8 @@
 
             this.organizationUnitService.deleteSelectedRegistrations(this.organizationUuid, this.unitUuid, request)
                 .then(() => {
+                    this.stateParameters.registrationsChanged();
+
                     this.options.refreshData();
```

Comment:
we have to talk about this line of code in a call.. it's hiding a promise making your state management rubbish :'(